starters/fastapi-safir-uws/values.yaml

```diff
@@ -0,0 +1,232 @@
+  affinity: {}
+
+  # -- Node selector rules for the <CHARTNAME> frontend pod
+  nodeSelector: {}
+
+  # -- Annotations for the <CHARTNAME> frontend pod
+  podAnnotations: {}
+
+  # -- Tolerations for the <CHARTNAME> frontend pod
+  tolerations: []
+
+cloudsql:
+  # -- Enable the Cloud SQL Auth Proxy sidecar, used with Cloud SQL databases
+  # on Google Cloud
+  enabled: false
+
+  image:
+    # -- Cloud SQL Auth Proxy image to use
+    repository: "gcr.io/cloudsql-docker/gce-proxy"
+
+    # -- Cloud SQL Auth Proxy tag to use
+    tag: "1.35.4"
+
+    # -- Pull policy for Cloud SQL Auth Proxy images
+    pullPolicy: "IfNotPresent"
+
+  # -- Instance connection name for a Cloud SQL PostgreSQL instance
+  # @default -- None, must be set if Cloud SQL is used
+  instanceConnectionName: null
+
+  # -- Resource limits and requests for the Cloud SQL Proxy container
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "100m"
+      memory: "20Mi"
+    requests:
+      cpu: "5m"
+      memory: "7Mi"
+
+cutoutWorker:
+  # -- Number of <CHARTNAME> worker pods to start
+  replicaCount: 2
+
+  image:
+    # -- Stack image to use for the backend worker
+    repository: "ghcr.io/lsst-sqre/<CHARTNAME>-worker"
+
+    # -- Tag of worker image to use
+    # @default -- The appVersion of the chart
+    tag: null
+
+    # -- Pull policy for workers
+    pullPolicy: "IfNotPresent"
+
+  # -- Resource limits and requests for the <CHARTNAME> worker pod
+  # @default -- See `values.yaml`
+  resources: {}
+
+  # -- Affinity rules for the <CHARTNAME> worker pod
+  affinity: {}
+
+  # -- Node selection rules for the <CHARTNAME> worker pod
+  nodeSelector: {}
+
+  # -- Annotations for the <CHARTNAME> worker pod
+  podAnnotations: {}
+
+  # -- Tolerations for the <CHARTNAME> worker pod
+  tolerations: []
+
+databaseWorker:
+  # -- Number of database worker pods to start
+  replicaCount: 1
+
+  # -- Resource limits and requests for the database worker pod
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "0.1"
+      memory: "200Mi"
+    requests:
+      cpu: "0.02"
+      memory: "85Mi"
+
+  # -- Affinity rules for the database worker pod
+  affinity: {}
+
+  # -- Node selection rules for the database worker pod
+  nodeSelector: {}
+
+  # -- Annotations for the database worker pod
+  podAnnotations: {}
+
+  # -- Tolerations for the database worker pod
+  tolerations: []
+
+redis:
+  config:
+    # -- Name of secret containing Redis password
+    secretName: "<CHARTNAME>"
+
+    # -- Key inside secret from which to get the Redis password (do not
+    # change)
+    secretKey: "redis-password"
+
+  persistence:
+    # -- Whether to persist Redis storage and thus tokens. Setting this to
+    # false will use `emptyDir` and reset all tokens on every restart. Only
+    # use this for a test deployment.
+    enabled: true
+
+    # -- Amount of persistent storage to request
+    size: "100Mi"
```

Comment:
I wonder if it's better to error on the side of being a bit larger since resizing PVCs is a pain. Up to you, though.

Reply:
I was going to say that the amount of data stored should be quite small since the largest thing stored here in this pattern is the arguments to the backend, but the vo-cutouts data is about 50MiB, so maybe it will be larger than I expect? Although that's also very surprising to me; that's way more data than I was expecting. I'll dig into this a bit.

Reply:
Aren't task outputs also stored here, including exception traces?

Reply:
Outputs are not, only a URL. Exception traces are, but only for a day.

It looks like Redis is not periodically cleaning up the appendonly database for some reason. I may be missing some command-line flags in our Redis chart. When I manually ran `BGREWRITEAOF`, the size of the vo-cutouts Redis database went down to about 1KB.